lib/features/auto-update-subscriptions.js

```diff
@@ -0,0 +1,29 @@
+module.exports = ({rtm, filterFn = () => Promise.resolve(true)}) =>
+  Promise.all([
+    rtm.on('subscription-created', subscription => {
+      if (subscription.allowRead !== true) return;
+
+      filterFn(subscription).then(ok => {
+        if (ok) {
+          rtm.subscribe({channel: subscription.channelId});
+        }
+      });
+    }),
+    rtm.on('subscription-deleted', subscription =>
+      filterFn(subscription).then(ok => {
+        if (ok) {
+          rtm.unsubscribe({channel: subscription.channelId});
+        }
+      })
```

Comment:
Keeping a dead subscription alive doesn't make sense. Broker will kill it later anyway.